apps/metadata-editor/src/app/dashboard/search-filters-summary-item/search-filters-summary-item.component.html

```diff
@@ -0,0 +1,22 @@
+<div *ngIf="(fieldValues$ | async)?.length > 0">
+  <div *ngIf="fieldType === 'dateRange'; else valuesBadge">
+    <span translate>search.filters.summaryLabel.{{ fieldName }}</span>
+    <gn-ui-badge
+      #valuesBadge
+      *ngFor="let fieldValue of dateRange$ | async"
+      [removable]="true"
+      (badgeRemoveClicked)="removeFilterValue(fieldValue)"
+      >{{ fieldValue['start'] }} - >{{ fieldValue['end'] }}</gn-ui-badge
```

Comment:
This can probably be encapsulated in a "getReadableValue" method to avoid having a ngIf in the template

Reply:
Good point, I've added f4d10b643e1aa1ea38f536670377926b949bbd18 for this.